.github/scripts/pack_macos.sh

```diff
@@ -48,6 +48,8 @@ pack_macos() {
 #  then
     import_certificate
     npx oclif-dev pack:macos
+    cd dist/macos
```

Comment:
This is only needed in case of draft releases. It will impact the usual releases from main. I guess we should take it up in a separate ticket to minimize impact.